FiveCalls/FiveCalls/IssueDetail.swift

```diff
@@ -38,11 +38,12 @@ struct IssueDetail: View {
                     RoundedRectangle(cornerRadius: 10)
                         .foregroundColor(Color(red: 0.85, green: 0.85, blue: 0.85))
                 }.padding(.bottom, 16)
-                NavigationLink(destination: IssueContactDetail(issue: issue, remainingContacts: issue.contactsForIssue(allContacts: store.state.contacts))) {
-                    PrimaryButton(title: "See your script", systemImageName: "megaphone.fill")
+                NavigationLink(value: IssueDetailNavModel(issue: issue, contacts: issue.contactsForIssue(allContacts: store.state.contacts))) {
```

Comment:
nit: I wonder if we need to pass the main app store into this view, it looks like we're only using it to access the contacts, but not performing any actions. We could probably just pass in the contacts to this view

Reply:
I don't quite understand; issue detail only gets an issue and an array of the remaining contacts, they are filtered by the `contactsForIssue` method which matches on area.

Reply:
Right, that's what I mean, we include the main store on line 12 of this file, but if we're only using it to get the contacts from the state, we could just include pass the contacts into the initializer for this view, unless you think we'll need access to the full store in the future. Just being overly picky here if we wanted to limit access is all.

Reply:
ok, I get it. Yeah, I'll give that a try